index.js

```diff
@@ -351,31 +357,21 @@ function delay(ms) {
  * Given an error from Got, see if Shopify told us how long to wait before
  * retrying.
  *
- * @param {Object} error Error object from Got call
+ * @param {Object} retryObject Got's input for the retry logic
  * @return {(Number|null)} The duration in ms, or `null`
  * @private
  **/
-function maybeRetryMS(error) {
-  // For simplicity, retry network connectivity issues after a hardcoded 1s.
-  if (retryableErrorCodes.has(error.code)) {
-    return 1000;
-  }
-
+function maybeRetryMS(retryObject) {
+  const { error, computedValue } = retryObject;
   const response = error.response;
 
-  if (response.headers && response.headers['retry-after']) {
-    return response.headers['retry-after'] * 1000 || null;
-  }
-
-  if (retryableStatusCodes.has(response.statusCode)) {
```

Comment:
This check shouldn't be needed anymore if we pass the same retryable status codes to `got` that we have defined in `shopify-api-node`